src/sdam/monitor.ts

```diff
@@ -329,6 +363,7 @@ function checkServer(monitor: Monitor, callback: Callback<Document | null>) {
 
     if (isAwaitable && monitor.rttPinger == null) {
       monitor.rttPinger = new RTTPinger(
+        monitor,
         monitor[kCancellationToken],
```

Comment:
Could this ctor just take the monitor now rather than passing these in seperately? 

I don't feel strongly about this, just clean up.